requirements-ci.txt

```diff
@@ -5,5 +5,5 @@ nbmake
 pymatgen
 pymatgen-analysis-diffusion
 mp-api
-# BoltzTraP2  # Does not work because numpy not detected.
```

Comment:
This was related to a build time isolation issue from `BoltzTraP2` side, have been fixed now